storage/fuzz/fuzz_targets/archive_operations.rs

```diff
@@ -0,0 +1,247 @@
+#![no_main]
+
+use arbitrary::Arbitrary;
+use commonware_runtime::{deterministic, Runner};
+use commonware_storage::{
+    archive::{Archive, Config, Identifier},
+    index::{translator::EightCap, Translator},
+};
+use commonware_utils::array::FixedBytes;
+use libfuzzer_sys::fuzz_target;
+
+type Key = FixedBytes<16>;
+type Value = FixedBytes<32>;
+type RawKey = [u8; 16];
+type RawValue = [u8; 32];
+
+#[derive(Arbitrary, Debug, Clone, PartialEq)]
+enum ArchiveOperation {
+    Put {
+        index: u64,
+        key_data: RawKey,
+        value_data: RawValue,
+    },
+    GetByIndex(u64),
+    GetByKey(RawKey),
+    HasByKey(RawKey),
+    Prune(u64),
+    Sync,
+    NextGap {
+        start: u64,
+    },
+}
+
+#[derive(Arbitrary, Debug)]
+struct FuzzInput {
+    operations: Vec<ArchiveOperation>,
+}
+
+fn fuzz(data: FuzzInput) {
+    if data.operations.is_empty() || data.operations.len() > 106 {
+        return;
+    }
+    let runner = deterministic::Runner::default();
+
+    runner.start(|context| async move {
+        let cfg = Config {
+            partition: "test".into(),
+            section_mask: 0xffff_ffff_ffff_ff00u64,
+            pending_writes: 1000, // Flush after 1000 writes
```

Comment:
In the next release, this implicit sync will be removed. We can cross that bridge when we get there I suppose lol.